pkg/bucketclass/bucketclass.go

```diff
@@ -100,7 +100,15 @@ func RunCreate(cmd *cobra.Command, args []string) {
 	}
 	name := args[0]
 
+	validPlacements := map[string]bool{
```

Comment:
This is really unneeded - for two values like this I would just prefer to write them explicitly in the condition.